source/includes/_xmedia.md

```diff
@@ -1,32 +1,34 @@
 ### **XMO Metrics**
-<a name="xmedia"></a>
+<a name="xmo_metrics"></a>
 
 ### Resource Overview
 
 | Method | URI Format |
 |---|---|
 | GET | /client_reports/xmedia/[gmaid]?[query_params] |
-#### API Name: xmedia
-### Usage
-Use GET to retrieve information for the Xmedia API that will retrieve metrics on xmedia campaigns. Data can be returned for a GMAID for a specific date range determined by start_date and end_date.  Data can also be returned in specified intervals by using the interval_size param. The requirements for these parameters are described below.
+
+Use GET to retrieve campaign performance metrics for the Cross Media Optimization (XMO) campaigns. 
+
+The data returned will include impressions, leads, clicks, calls, qualified_web_events, web_events, emails, chats and totals of the leads. Please note chats will be reported as 0 in this API. Please use the XMO channel metrics API to get the chat counts. The total leads are based on all web events.
+
 
 ### Parameters
 
 When using the GET method, the results can be filtered using these parameters:
 
-| Param | Required | Default | Description |
-|---|---|---|---|
-|`start_date`| yes |--|Restricts the results to those ocurring on or after this date.|
-|`end_date`| yes |--|Restricts the results to those ocurring on or before this date.|
-|`global_master_campaign_id[]`| no |--|Restrict results to one or more specific campaigns|
-|`campaign_status[]`| no |running|Restrict results to all campaigns with given status values.  Allowed values are `running`, `stopped` and `ended`|
-|`campaign_cycle`| no |--|Restrict results to a single campaign cycle|
-|`interval_size`| no | day | Use `calendar_month` or `calendar_week` to roll up the data points into calendar intervals (default is 1 day per interval)|
-|`include_cycles`| no | false |Set to true or false on whether to include cycle nesting.  Default value is false.|
-|`markup_type`| no |--|Only valid option is "percentage".|
-|`markup_value`| no | 0 |When `markup_type` is "percentage" this is the percent markup.|
-
-### Examples:
+| Param | Function|
+|---|---|
+|`start_date`|Restricts the results to those ocurring on or after this date.|
+|`end_date`|Restricts the results to those ocurring on or before this date.|
+|`global_master_campaign_id[]`|Restrict results to one or more specific campaigns. This should be a comma separated string. Ex: global_master_campaign_id[]=TEST_1,TEST_2|
```

Comment:
this is incorrect, how array request parameters are specified depends on the framework or tool making the request.  I suggest you leave the second sentence out.

Reply:
Updated